stl/inc/inplace_vector

```diff
@@ -0,0 +1,1560 @@
+// inplace_vector standard header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef _INPLACE_VECTOR_
+#define _INPLACE_VECTOR_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+#include <xmemory>

```

Comment:
The following contents should be guarded by `_HAS_CXX26`, and we should emit some message in pre-C++26 modes, see also `<span>`.

It's perhaps fine to allow `<inplace_vector>` in C++23 mode for the purpose of CI at this moment.